tests/test_report.py

```diff
@@ -0,0 +1,90 @@
+class TestQueryCount(object):
+    """Test report generated by --querycount parameter"""
+
+    def test_querycount_report_header(self, django_testdir):
+        django_testdir.create_test_module('''
+            def test_zero_queries():
+                pass
+        ''')
+
+        result = django_testdir.runpytest_subprocess('--querycount=5')
+        result.stdout.fnmatch_lines([
+            '*== top 5 tests with most queries ==*'
```

Comment:
I intended just to test the headers here. Do you think I should add tests to the report lines here or [the updated version of the other test case](https://github.com/pytest-dev/pytest-django/pull/412/files#diff-140a9be78461cc0081b195a71b6f08f1R24) is enough?